github/commands/pull_request.py

```diff
@@ -79,13 +79,13 @@ def on_select_action(self, idx):
         elif idx == 1:
             show_single_line_input_panel(
                 "Enter branch name for PR {}:".format(self.pr["number"]),
-                "{}/{}".format(self.pr["user"]["login"], self.pr["head"]["ref"]),
```

Comment:
also change the default value of branch name from owner/repo to owner-repo to avoid ambiguity.